### PR TITLE
Add support for hover pointer events

### PR DESCRIPTION
**Pull request by abarth**
These are implemented on macOS and Fuchsia.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: content_handler/runtime_holder.cc [hunk 1]
```diff
@@ -269,6 +269,25 @@ void RuntimeHolder::OnEvent(mozart::EventPtr event,
     pointer_data.physical_x = event->pointer_data->x;
     pointer_data.physical_y = event->pointer_data->y;
 
+    switch (pointer_data.change) {
+      case blink::PointerData::Change::kDown:
+        down_pointers_.insert(pointer_data.pointer);
+        break;
+      case blink::PointerData::Change::kCancel:
+      case blink::PointerData::Change::kUp:
+        down_pointers_.erase(pointer_data.pointer);
```
Comment (Hixie) [inline content_handler/runtime_holder.cc]: can the C++ code miss the down event somehow? The platforms guarantee that we won't, right?

**Comment by Hixie** [comment]
LGTM

File: content_handler/runtime_holder.cc (hunk 1 above)
Comment (abarth) [inline content_handler/runtime_holder.cc]: We're not supposed to miss events.

**Pull request merged**